debian/changelog

```diff
@@ -1,4 +1,4 @@
-oi-bus (0.2.3) UNRELEASED; urgency=medium
+oi-bus (0.2.3) unstable; urgency=medium
```

Comment:
I thought this is supposed to be done by the person pushing the built package to our internal debian repo?